microya/src/androidTest/java/com/papershift/microya/supporting/ImgurResponse.kt

```diff
@@ -0,0 +1,15 @@
+package com.papershift.microya.supporting
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ImgurResponse<T>(
```

Comment:
A more consistent name here would be `ImgurDataResponse` (to be in line with `ImgurErrorResponse`).

```suggestion
data class ImgurDataResponse<T>(
```